src/services/kpis/kpis.svc.coffee

```diff
@@ -76,15 +76,15 @@ angular
       unless _self.locked
         _self.locked = true
 
-        return ImpacDashboardsSvc.load(force).then(
-          ->
+        return $q.all([ImpacMainSvc.loadUserData(force), ImpacDashboardsSvc.load(force)]).then(
+          (results)->
             if _.isEmpty(_self.getKpisTemplates()) || force
 
               # clear array
               _.remove _self.config.kpisTemplates, (-> true)
 
               orgUids = _.pluck _self.getCurrentDashboard().data_sources, 'uid'
-              ssoSessionId = _self.getSsoSessionId()
+              ssoSessionId = results[0].sso_session
```

Comment:
particular reason for this one? may break if somebody append another promise in the future...